crates/bevy_asset/src/server/info.rs

```diff
@@ -282,7 +282,7 @@ impl AssetInfos {
 
     pub(crate) fn get_path_and_type_id_handle(
         &self,
-        path: &AssetPath,
+        path: &AssetPath<'_>,
```

Comment:
```suggestion
        path: &AssetPath,
```
Can leave this without explicitly specifying lifetimes, since the only generics here are the lifetimes, they'll get auto elided. I tested this locally, it compiles.

Reply:
I prefer the style with `<'_>`, and it could even be enforced in CI if we want. See https://doc.rust-lang.org/rustc/lints/listing/allowed-by-default.html#elided-lifetimes-in-paths

Reply:
Fair enough, I can see arguments for both removing and keeping this.